data/companies/alma/corporate.yaml

```diff
@@ -0,0 +1,22 @@
+---
+type: corporate
+name: Alma Media
+template: publisher
+comment: methodology question - if we are excluding print emissions, how do we exclude print employees?
```

Comment:
Some sustainability reports break out print from digital (see Guardian, Axel Springer as examples). If it's not broken out, we could use the % of print revenue as a starting point. Alternatively, we could throw out the report entirely since we can't get the detail we need.